.pipelines/nuget-publishing.yml

```diff
@@ -42,19 +47,22 @@ parameters:
   - '12.2'
   default: '11.8'
 
-- name: publish_to_ado_feed
-  displayName: 'Publish to Azure DevOps Feed'
-  type: boolean
-  default: false
-
 resources:
   repositories:
   - repository: manylinux
     type: Github
     endpoint: Microsoft
     name: pypa/manylinux
     ref: 5eda9aded5462201e6310105728d33016e637ea7
-
+  pipelines:
```

Comment:
Remove this trigger?  Even if the source is not changed, we still need to run this pipeline.

Reply:
https://learn.microsoft.com/en-us/azure/devops/pipelines/process/pipeline-triggers?view=azure-devops&tabs=yaml#branch-considerations

Reply:
Removed